functional_tests/features/bundle_ui.feature

```diff
@@ -0,0 +1,133 @@
+#           | 1                 | Publishing Officer         | Publishing Admin    | {"Role": "Publishing Officer", "Creator Role": "Publishing Admin",   "status": "Draft", "preview_teams": false, "add_rel_cal": false, "add_stat_page": false} |
+#           | 1                 | Publishing Officer         | Publishing Officer  | {"Role": "Publishing Officer", "Creator Role": "Publishing Officer", "status": "Draft", "preview_teams": false, "add_rel_cal": false, "add_stat_page": false} |
+
+
+#---- Bundle Preview UI Tests -----
+
+  Scenario Outline: A User can preview a bundle
+        Given there is a <Role> user
+        And there is a <Creator Role> user
+        And there are 1 Statistical Analysis pages
+        And there are 1 release calendar pages
+        And there is a preview team
+        And the <Role> is a member of the preview team
+        And there are <number_of_bundles> bundles with <Bundle_Details>
+        When the <Role> logs in
+        Then the user can see the Bundles ready for preview
+        Then the <Role> can preview a bundle
+
+      Examples: bundles
+           | number_of_bundles | Role                       | Creator Role        | Bundle_Details                                                                                                                                              |
+           | 1                 | Publishing Admin           | Publishing Admin    | {"Role": "Publishing Admin",   "Creator Role": "Publishing Admin",   "status": "In_Review", "preview_teams": true,  "add_rel_cal": true, "add_stat_page": true} |
+           | 1                 | Publishing Officer         | Publishing Admin    | {"Role": "Publishing Officer", "Creator Role": "Publishing Admin",   "status": "In_Review", "preview_teams": true,  "add_rel_cal": true, "add_stat_page": true} |
+           | 1                 | Viewer                     | Publishing Admin    | {"Role": "Publishing Admin",   "Creator Role": "Publishing Admin",   "status": "In_Review", "preview_teams": true,  "add_rel_cal": true, "add_stat_page": true} |
+           | 1                 | Publishing Admin           | Publishing Officer  | {"Role": "Publishing Admin",   "Creator Role": "Publishing Officer", "status": "In_Review", "preview_teams": true,  "add_rel_cal": true, "add_stat_page": true} |
+           | 1                 | Publishing Officer         | Publishing Officer  | {"Role": "Publishing Officer", "Creator Role": "Publishing Officer", "status": "In_Review", "preview_teams": true,  "add_rel_cal": true, "add_stat_page": true} |
+           | 1                 | Viewer                     | Publishing Officer  | {"Role": "Viewer",             "Creator Role": "Publishing Officer", "status": "In_Review", "preview_teams": true,  "add_rel_cal": true, "add_stat_page": true} |
+
+
+      Scenario Outline: A User cannot preview a bundle
```

Comment:
Also similar to some other scenarios above, it doesn't seem to be doing anything?